File: java-suite/src/main/java/de/fraunhofer/iem/classesAndObjects/interfaceInJava/interfaceInheritance/Movable.java
package de.fraunhofer.iem.classesAndObjects.interfaceInJava.interfaceInheritance;

/**
 * Demonstrates the multiple inheritance in interface
 *
 * @author Ranjith Krishnamurthy
 */
public interface Movable extends Runnable, Walkable {
    public void jog();
}

File: kotlin-suite/src/main/kotlin/de/fraunhofer/iem/classesAndObjects/interfaceInKotlin/interfaceInheritance/Movable.kt
package de.fraunhofer.iem.classesAndObjects.interfaceInKotlin.interfaceInheritance

/**
 * Demonstrates the multiple inheritance in interface
 *
 * @author Ranjith Krishnamurthy
 */
interface Movable : Runnable, Walkable {
    fun jog()
}